parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -188,14 +188,13 @@ renderTypeError e env src = case e of
           <> " has to be"
 
   ExistentialMismatch {..} -> mconcat
-    [ Pr.wrap $ mconcat
-        [ preamble
-        , " "
-        , "Here, one is "
-        , style Type1 (renderType' env expectedType)
-        , " and another is "
-        , style Type2 (renderType' env foundType)
-        , ":"]
+    [ Pr.lines
+        [ Pr.wrap preamble
+        , ""
+        , "Here, one   is:  " <> style Type1 (renderType' env expectedType)
+        , "and another is:  " <> style Type2 (renderType' env foundType)
+        , ""
```

Comment:
Minor suggestion to put the alignment padding after the `:` rather than aligning the `is`.

Reply:
Here they are for comparison -

```
Here, one is:    Foo x (Qux j Bat)
and another is:  Foo x (Qux g Nat)
```

```
Here,  one  is:  Foo x (Qux j Bat)
and another is:  Foo x (Qux g Nat)
```

My thinking was the second one draws focus to the aligned edge better, and the jagged edge is a little distracting. Wdyt? 

@hojberg You might have an opinion on this

Reply:
Hm, yeah, thanks for the side-by-side; I'm not sure.  They both look weird. 😅  I don't mind it staying as-is until there's a stronger opinion.

Reply:
A third option is like in the very next block:
```
  Here, one is:  Foo x (Qux j Bat)
and another is:  Foo x (Qux g Nat)
```

Reply:
I think I like your third option, will go with that!

Reply:
Disregard, the third option looks really weird in context, because that left edge should be aligned with the paragraph above it. 

The other example indents this portion of the message. There was some reason I didn't go with that here. I'm just going to leave as is for now, but am open to doing a more cohesive pass through the error message formatting.